### PR TITLE
Zebra.ZebraZCCardPrinter version 1.05.00

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107473)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with:
> image

I’ve seen that one before, in that J5Create installer, flags are /S

File: manifests/z/Zebra/ZebraZCCardPrinter/1.05.00/Zebra.ZebraZCCardPrinter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Zebra.ZebraZCCardPrinter
+PackageVersion: 1.05.00
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/ZC100-300_driver.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 901B62DF401C9E3A6314E19704492381D0104C739E86CD54F294C8883053238B
```
Comment (stephengillie) [inline manifests/z/Zebra/ZebraZCCardPrinter/1.05.00/Zebra.ZebraZCCardPrinter.installer.yaml:10]:
```suggestion
  InstallerSha256: 901B62DF401C9E3A6314E19704492381D0104C739E86CD54F294C8883053238B
  InstallerSwitches:
    Silent: /S
```

**Comment by stephengillie** [comment]
With the installer switch in place, installation failed with:
> 2023-05-22 16:38:27.127 [CLI ] ShellExecute installer failed: 2147753984

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

Works in interactive-only mode (installs to `C:\Program Files (x86)\Zebra Technologies\ZC Printer Drivers` but errors out in silent mode.

**Pull request closed without merging**